WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -222,7 +224,8 @@ private static int purgePostsForTag(SQLiteDatabase db, ReaderTag tag) {
                        + "tag_type=? ORDER BY " + getSortColumnForTag(tag) + " DESC LIMIT ?)";
         int numDeleted = db.delete("tbl_posts", where, args);
         AppLog.d(AppLog.T.READER,
-                 String.format("reader post table > purged %d posts in tag %s", numDeleted, tag.getTagNameForLog()));
+                 String.format(Locale.ENGLISH,
+                               "reader post table > purged %d posts in tag %s", numDeleted, tag.getTagNameForLog()));
```

Comment:
In light of [removing the call parameters alignment](https://github.com/wordpress-mobile/WordPress-Android/pull/7388#issuecomment-371436745), wdyt about holding back all the changed in this PR that are not Locale related and just re-indent code?

Reply:
I think this will push the line width over 120 characters and will fail the build :(

Reply:
We'll respect the 120chars limit too but avoid indenting the wrapped call params that much. What am I missing? 🤔